images/prometheus-cloudwatch-exporter/config/main.tf

```diff
@@ -0,0 +1,19 @@
+terraform {
+  required_providers {
+    apko = { source = "chainguard-dev/apko" }
+  }
+}
+
+variable "extra_packages" {
+  description = "The additional packages to install"
+  default     = ["cloudwatch-exporter"]
+}
+
+data "apko_config" "this" {
+  config_contents = file("${path.module}/template.apko.yaml")
```

Comment:
If `template.apko.yaml` is moved to `latest.apko.yaml` then this should also be changed.

Reply:
Done!